.pre-commit-config.yaml

```diff
@@ -0,0 +1,54 @@
+# pre-commit package installation is necessary to use pre-commit.
+# $ pip install pre-commit
+# $ pre-commit install
+
+default_language_version:
+  python: python3
+
+repos:
+  # Args are based on setup.cfg.
+  - repo: https://github.com/psf/black
+    rev: 24.8.0
+    hooks:
+      - id: black
+        exclude: .eggs|.git|.hg|.mypy_cache|.venv|venv|_build|buck-out|build|dist|docs
```

Comment:
```suggestion
        exclude: docs
```
It's okay not to pass the files ignored by .gitignore, because `pre-commit` only passes the files that are staged.

Reply:
And even this `exclude:` statement is removable to use the `force-exclude` option in `pyproject.toml` (see https://github.com/psf/black/issues/3013).

Reply:
Thanks for the comment! I will add `force-exclude` option in `pyproject.toml`.